chess engine/ponto_virada.py

```diff
@@ -0,0 +1,14 @@
+
+def pontosVirada (scores):
+    pontos_virada = [0]
+
+    for i in range(1, len(scores)):
+        if type(scores[i]) == int and type(scores[i-1]) == int:
+            if abs(scores[i]) >= (abs(scores[i - 1]) + 150) and abs(scores[i - 1]) < 150:
+                pontos_virada.append(1)
+            else:
+                pontos_virada.append(0)
+        else:
```

Comment:
this condition should be possible?

Reply:
made a comment on the code to address this